composer.json

```diff
@@ -55,6 +55,7 @@
         "drupal/color_field": "^2.0",
         "drupal/config_ignore": "^2.2",
         "drupal/config_import_locale": "^2.0",
+        "drupal/content_fixtures": "^3.1",
```

Comment:
`drupal/content_fixtures ` should be a `dev` dependency and probably added as a dependency in a module requiring (cf. https://github.com/itk-dev/os2loop/blob/develop/composer.json#L21-L24 and https://github.com/itk-dev/os2loop/blob/develop/web/profiles/custom/os2loop/modules/os2loop_fixtures/composer.json).

Reply:
Changed